### PR TITLE
RT-539:Update to Angular 15, 15.1.1

### PR DESCRIPTION
**Pull request by yashpmohod**
## Overview
Updated angular to new minor version 15.1.1.
Fixed all UI Glitch.
 

## How it was tested
Tested by running app on local.

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [x] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

File: projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.css [hunk 1]
```diff
@@ -2,6 +2,7 @@
   display: flex;
   flex-flow: column;
   height: 100vh;
+  padding-left: 2vh;
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.css:5]: why is padding-left based on the viewport height?

File: projects/ngx-charts-on-fhir/src/lib/data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component.css [hunk 2]
```diff
@@ -3,6 +3,7 @@
   background-color: #3f51b5;
   position: absolute;
   height: 100vh;
-  width: 48px;
+  width: 52px;
   padding-top: 16px;
+  z-index: 999;
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component.css:8]: Changing `z-index` is not the best solution. It's better to avoid overlapping elements in the first place. Please change the `left` property of `mat-sidenav-container` in `app.component.scss` to match the toolbar width.

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css [hunk 3]
```diff
@@ -10,7 +10,7 @@
 .scroll-container {
   height: 100vh;
   overflow-y: auto;
-  padding: 8px;
+  padding: 8px 8px 8px 12px;
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css:13]: I think you can revert this change after you fix mat-sidenav-container

**Review by semicolin — CHANGES_REQUESTED** [review]

File: projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.css (hunk 1 above)
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.css:5]: @semicolin, try and run i have added this, we don't need this removing...

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css (hunk 3 above)
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css:13]: Yes, you are right.

**Review by semicolin — APPROVED** [review]
Awesome!
